packages/docs/src/pages/products/index.md

```diff
@@ -1,76 +1,83 @@
+---
+sidebar_label: Medplum Products
+sidebar_position: 1
+---
+
 # Products
 
 At Medplum our goal is to help you build a healthcare application quickly, easily and compliantly.
 
 :::caution Note
 
-This section is under construction.  Check back frequently for updates.
+This section is under construction. Check back frequently for updates.
 
 :::
 
 ## Features
 
+Medplum supports building applications with the following features. Reference implementations can be found in our [repository](https://github.com/medplum) and on our reference apps [Foo Medical](https://foomedical.com/) and [Provider Application](https://provider.foomedical.com/).
+
 ### Forms
 
-Create custom forms, surveys, and questionnaires quickly to enable engagement, assessment, patient onboarding, provider workflow and retention.
+Create custom forms, surveys, and questionnaires quickly to enable engagement, assessment, patient onboarding, provider workflow and retention. [Learn More](../products/forms)
```

Comment:
@rahul1 - because I had to link this way, I suspect Docusaurus is misconfigured somehow, but am not sure.  As these are in the same directory, I expected [Learn More](forms) would have been sufficient.  Am I mistaken?